### ISSUE TITLE
Summary.py > save 's export the output

### ISSUE THREAD

**Issue by scene-the-ella** [YufeiWang777/LRRU#14]
![image](https://github.com/YufeiWang777/LRRU/assets/16212874/583e9507-e47e-4738-a801-c5c6edc4fa46)

Hi! Thank you for your nice work!
BTW, isn't it` pred = output['results'][-1].detach() ` instead of `pred = output[self.args.output][-1].detach()`?
Cuz that line causes KeyError.

**Comment by jisoo0-0** [comment]
In my reproduction, that was not the problem. There was a key "results" in the output dictionary.

**Comment by scene-the-ella** [comment]
> In my reproduction, that was not the problem. There was a key "results" in the output dictionary.
Hi @jisoo0-0 ! Thank you for your appreciation of my issue in responding on behalf of the author! However, It seems that what you're saying now is exactly the same as my initial question. I'm actually arguing that `output['results'][-1]` is correct, not `output[self.args.output][-1]`. 😉

**Comment by scene-the-ella** [comment]
> In my reproduction, that was not the problem. There was a key "results" in the output dictionary.

Hi @jisoo0-0 ! Thank you for your appreciation of my issue in responding on behalf of the author! However, It seems that what you're saying now is exactly the same as my initial question. I'm actually arguing that `output['results'][-1]` is correct, not `output[self.args.output][-1]`. 😉

**Comment by YufeiWang777** [comment]
Hi, you can change it to the key you want.
